### DISCUSSION TITLE
Bug Fixes: some suggestions

### DISCUSSION THREAD

**Discussion by polaris911** [OnionUI/Onion#556] [General]
First off, thank you for making Onion OS! Some of these suggestions are probably already on your radar. 

1. On the Consoles page, allow the triggers to move page left and right. 

2. With Onion Launcher mode, if you're in the menus when you shut off the handheld, when you turn it back on, it restarts in the middle of the game. It should resume in the menus. 

3. With Onion Launcher mode, the screen brightness is not remembered when you turn the handheld on. You have to go into the menus first, then it will adjust. I saw a note this was fixed in 3.8.3, but it still happens.

4. Reduce clicks and sound pops when entering/exiting Retroarch. 

5. The Favorites list shows newest additions at the bottom, but Recents list shows most recet at the top. They should be consistent and both show newest things at the top. 

6. In Favorites list, favorited Neo Geo games don't show the title. It just says "Neo Geo"

7. In the Recents list, things like "Open" and "Onion Installer" show up. I think these should not show up here.

8. The "Refresh Roms" option should be available in Settings and within the individual Consoles listed. It can only be accessed from the main menu by pressing the Menu button and no where else.  

9. The In-Game menu could be improved to be more like the original firmware. The menu is small and hard to read. The screen momentarily turns black while opening this menu which is disorienting. On some consoles, the Menu button just quits the game and doesn't summon the menu.

**Comment by Sichroteph** [comment]
I am glad to report that the brightness settings should be universally synced on 3.10.5

**Comment by Sichroteph** [comment]
There are great suggestions here, thank you
